aicsimageio/readers/ome_tiff_reader.py

```diff
@@ -101,6 +102,15 @@ def _is_supported_image(
             log.error(e)
             return False
 
+        # cant connect to external schema resource (no internet conection)
+        except URLError as e:
+            log.error(
+                "Could not validate OME XML against referenced schema "
+                "(no internet connection)"
+            )
+            log.error(e)
```

Comment:
I might recommend combining both of these log messages into a single call. I could imagine some scenarios where these two calls might be interlaced with other messages and would become confusing, since the context of the error has been separated from the actual error itself.

Reply:
Totally agree. Combining!